src/dra/843B0.c

```diff
@@ -2498,7 +2498,355 @@ void EntityStopWatchExpandingCircle(Entity* self) {
 }
 
 // stopwatch subweapon effect. stops enemies (Dra Entity 0x2A)
-INCLUDE_ASM("dra/nonmatchings/843B0", EntityStopWatch);
+void EntityStopWatch(Entity* self) {
+    Primitive* prim;
+    s16 a;
+    s16 b;
+    s32 x;
+    s32 c;
+    s32 y;
+    s32 d;
+    s16 offsetX, offsetY;
+
+    if (g_unkGraphicsStruct.unk0) {
+        g_unkGraphicsStruct.D_800973FC = 0;
+        if (self->step && (self->step < 4)) {
+            self->step = 4;
+        }
+    }
+
+    switch (self->step) {
+    case 0:
+        if (g_unkGraphicsStruct.unk0) {
+            DestroyEntity(self);
+            return;
+        }
+        self->primIndex = AllocPrimitives(4, 3);
+        if (self->primIndex == -1) {
+            DestroyEntity(self);
+            return;
+        }
+        self->flags = 0x0C860000;
```

Comment:
Could you document this one with the flags from `game.h` at line `253`? For example `0x00800000` is `FLAG_HAS_PRIMS`. There are other places like `prim->drawMode` too